ariblake/bloom
# Product reviews

### Goal
Add product review app to store. 

### Deployment Instructions
- Pull master
- ensure that your production id is correct in `config.yml`
- `theme deploy --env production`

### Testing Instructions
- [x] Go to the [PDP](https://cwh7asyry7tej32v-31795118219.shopifypreview.com/products/clay-plant-pot)
   - Scroll down and write a review
   - Refresh the page and ensure that the review is showing
   - Scroll to the top of the page, ensure that the reviews badge is showing near the price

- [x] Go to the [Collection Page](https://cwh7asyry7tej32v-31795118219.shopifypreview.com/collections/all)
   - Ensure that the badges are showing for each product that has a review
   - Ensure that products with no reviews don't have a review badge

### Tested On
Chrome on Windows

### Code Request Quality Checklist


### Reviewers
@hillary25
